docs/tutorials/pwm_control_flight_controller.rst

```diff
@@ -363,53 +390,79 @@ Reboot the flight controller and connect to it again. This time in the "Messages
 
-1. Start with the motor powered off, but connected to the flight controller.
+1. Start with the module powered off, but connected to the flight controller.
 2. Select "Setup" from the top toolbar, and then expand "Optional Hardware" on the sidebar
-3. Select "Motor Test" from the "Optional Hardware" options.
-4. Power on the motor and wait for it to complete its startup song.
+3. Select "Motor Test" from the "Optional Hardware" options, see the image below as an example of what the Motor Test screen should look like.
+
+  .. figure:: ../_static/tutorial_images/pwm_flight_controller/mp_motor_test.png
+    :align: center
+
+    Motor Test Screen in Mission Planner
+
+4. Power on the module and wait for it to complete its startup song.
+
+  * If you have disabled the safety switch and are using Standard PWM, the module should play its arming song after startup when attached to the flight controller.
+
 5. Arm your safety switch if you have one.
    
-  * If using PWM, the motor should play its two-note arming song as the flight controller starts sending 0% throttle commands once the safety switch is armed
-  * On DSHOT, the motor will not arm yet.
+  * If using Standard PWM, the module should play its two-note arming song as the flight controller starts sending 0% throttle commands once the safety switch is armed
+  * On DSHOT, the module will not arm yet.
   
-6. Set the "Throttle %" to 3% and the "Duration" to 5s, and click "Test All Motors" 
-7. The motor should spin slowly for 5 seconds, and then stop.
+6. Set the "Throttle %" to 5% and the "Duration" to 5s, and click "Test All Motors" 
+7. The module should spin slowly for 5 seconds, and then stop.
    
-  * On PWM, the motor will spin and stop without any additional arming or disarming noises
-  * On DSHOT, the motor will arm when you send the command, spin for the duration, and then stop and disarm again, since DSHOT sends a specific disarm command at the end of the test.
+  * On Standard PWM, the module will spin and stop without any additional arming or disarming noises
+  * On DSHOT, the module will arm when you send the command, spin for the duration, and then stop and disarm again, since DSHOT sends a specific disarm command at the end of the test.
    
-8. Try some other throttle levels to see the motor running at different speeds
+8. Try some other throttle levels to see the module running at different speeds
    
-  * For DSHOT, you cannot go straight to a high throttle level, as the motor will disarm after each command, and only re-arms on a throttle command close to 0%. So for DSHOT testing
-    increase the duration and send a low level throttle command to arm the motor, and before that command ends and the motor disarms, send another higher throttle command.  
+  * For DSHOT, you cannot go straight to a high throttle level, as the module will disarm after each command, and only re-arms on a throttle command close to 0%. So for DSHOT testing
```

Comment:
For DSHOT, you cannot command a high throttle level immediately as the module will...